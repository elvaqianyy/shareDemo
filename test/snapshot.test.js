import Snapshot from '../lib/snapshot.vue'
import { shallowMount } from '@vue/test-utils'

it('正确渲染', () => {
  const wrapper = shallowMount(Snapshot, {
    propsData: {
      href: 'https://www.baidu1.com',
      name: 'hello world'
    }
  })
  expect(wrapper).toMatchSnapshot()
})