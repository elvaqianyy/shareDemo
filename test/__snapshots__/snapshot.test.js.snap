// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`正确渲染 1`] = `<div id="nav" href="https://www.baidu1.com"><a>hello world</a></div>`;
